### DISCUSSION TITLE
OIDCUnAutzAction action to explain how to get authorized

### DISCUSSION THREAD

**Discussion by chrisinmtown** [OpenIDC/mod_auth_openidc#711] [Ideas]
We need to display a message to users who are authenticated by mod-auth-openidc via the company-wide OAuth server, but who are NOT specifically authorized for the protected webapp. The message should explain how to get authorized, which is usually an involved process and always difficult to discover.

I was thinking maybe this could be a new option for the OIDCUnAutzAction action. Some possibilities might be:
1) Display static text with a message (limiting, but better than nothing)
2) Redirect to a URI, which might be an unprotected page in the webapp or another company site with instructions

Thanks in advance for considering it.

**Comment by zandbelt** [comment]
that sounds like a nice enhancement of `OIDCUnAutzAction` indeed

**Reply by chrisinmtown** [reply]
Thanks :) I'm sure we're not the only ones with this need, hope you can find time for it.

**Reply by chrisinmtown** [reply]
@zandbelt There's this really tempting link at the right side of this page "Create issue from discussion" .. but I'm definitely an end user and that tracker says "not for end users", so please tell me the right workflow for this.

**Reply by zandbelt** [reply]
yes, you can add it as a feature request

**Reply by chrisinmtown** [reply]
Done!
